systems/aws-pcluster/system.py

```diff
@@ -17,21 +19,76 @@
         "sys_cores_per_node": 4,
         "sys_mem_per_node": 8,
     },
+    "hpc7a.48xlarge": {
+        "sys_cores_per_node": 96,
+        "sys_mem_per_node": 768,
+    },
+    "hpc6a.48xlarge": {
+        "sys_cores_per_node": 96,
+        "sys_mem_per_node": 384,
+    },
 }
 
 
-class Aws(System):
+class AwsPcluster(System):
     variant(
         "instance_type",
-        values=("c6g.xlarge", "c4.xlarge"),
+        values=("c6g.xlarge", "c4.xlarge", "hpc7a.48xlarge", "hpc6a.48xlarge"),
         default="c4.xlarge",
         description="AWS instance type",
     )
 
     def initialize(self):
         super().initialize()
-        self.scheduler = "mpi"
+        self.scheduler = "slurm"
         # TODO: for some reason I have to index to get value, even if multi=False
         attrs = id_to_resources.get(self.spec.variants["instance_type"][0])
         for k, v in attrs.items():
             setattr(self, k, v)
+
+    def system_specific_variables(self):
+        return {
+            "extra_cmd_opts": '--mpi=pmix --export=ALL,FI_EFA_USE_DEVICE_RDMA=1,FI_PROVIDER="efa",OMPI_MCA_mtl_base_verbose=100',
+        }
```

Comment:
Right now, this produces experiment scripts like:

```
#SBATCH --mpi=pmix --export=ALL,FI_EFA_USE_DEVICE_RDMA=1,FI_PROVIDER="efa",OMPI_MCA_mtl_base_verbose=100
#SBATCH -n 8
#SBATCH -N 1
...
srun --mpi=pmix --export=ALL,FI_EFA_USE_DEVICE_RDMA=1,FI_PROVIDER="efa",OMPI_MCA_mtl_base_verbose=100 -n 8 -N 1 saxpy -n 512 >> ...
```

(i.e. these are generated as sbatch opts and command opts) - this is decided by the allocation modifier (specifically for the SLURM scheduler). I think it's worth checking if these should be batch opts

* If so, this can be rewritten to generate a separate batch opt for `--mpi` and `--export`
* If not, I can change the allocation modifier

I don't think this decision need stop this PR, but the generated experiment scripts will have weird sbatch directives until that is decided.